tasks/main.yml

```diff
@@ -7,6 +7,15 @@
     name:
       - exim4
       - heirloom-mailx
+  when: ansible_distribution != "Debian" or (ansible_distribution == "Debian" and ansible_distribution_major_version < "10")
+
+- name: Install packages for mail sending
+  apt:
+    name:
+      - exim4
+      - s-nail
+  when: ansible_distribution == "Debian" and ansible_distribution_major_version >= "10"
```

Comment:
Besides the same issue as above, maybe it would make more sense to use this task when the distribution is not Debian since newer versions of Debian derivatives are likely to have that package instead of heirloom-mailx?